src/main/java/com/epam/reportportal/service/LaunchIdLock.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.reportportal.service;
+
+/**
+ * A service to perform blocking operation to get single launch UUID for multiple clients.
+ */
```

Comment:
Please, add yourself as an author in the following [way](https://github.com/reportportal/service-api/blob/e62ae95d2fb66d65221afdf13d1ef3d68d3603ee/src/main/java/com/epam/ta/reportportal/job/LaunchCleanerService.java#L25)

Reply:
Added